<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDbLoginComponent</class>
 <widget class="QFrame" name="CDbLoginComponent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>361</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DB login</string>
  </property>
  <layout class="QVBoxLayout" name="vl_LoginComponent">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tb_LoginToDbAndHints">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tb_LoginToDb">
      <attribute name="title">
       <string>Login to DB</string>
      </attribute>
      <layout class="QVBoxLayout" name="vl_LoginToDb">
       <property name="leftMargin">
        <number>4</number>
       </property>
       <property name="topMargin">
        <number>4</number>
       </property>
       <property name="rightMargin">
        <number>4</number>
       </property>
       <property name="bottomMargin">
        <number>4</number>
       </property>
       <item>
        <widget class="QStackedWidget" name="sw_LoginLogoff">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="pg_Login">
          <layout class="QVBoxLayout" name="vl_PageLogin">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QFrame" name="fr_Login">
             <layout class="QGridLayout" name="gl_Login">
              <property name="leftMargin">
               <number>4</number>
              </property>
              <property name="topMargin">
               <number>4</number>
              </property>
              <property name="rightMargin">
               <number>4</number>
              </property>
              <property name="bottomMargin">
               <number>4</number>
              </property>
              <property name="spacing">
               <number>6</number>
              </property>
              <item row="3" column="2">
               <widget class="QWidget" name="wi_Button" native="true">
                <layout class="QHBoxLayout" name="hl_LoginFrame">
                 <property name="leftMargin">
                  <number>2</number>
                 </property>
                 <property name="topMargin">
                  <number>2</number>
                 </property>
                 <property name="rightMargin">
                  <number>2</number>
                 </property>
                 <property name="bottomMargin">
                  <number>2</number>
                 </property>
                 <item alignment="Qt::AlignRight">
                  <widget class="QPushButton" name="pb_Login">
                   <property name="maximumSize">
                    <size>
                     <width>150</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>login</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="lbl_Database">
                <property name="text">
                 <string>Database:</string>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <spacer name="hs_Login">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Fixed</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>75</width>
                  <height>10</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="5" column="0">
               <spacer name="vs_FrameLogin">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="lbl_User">
                <property name="text">
                 <string>Name:</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="lbl_Password">
                <property name="text">
                 <string>Password:</string>
                </property>
               </widget>
              </item>
              <item row="2" column="2">
               <widget class="QLineEdit" name="le_Password">
                <property name="echoMode">
                 <enum>QLineEdit::PasswordEchoOnEdit</enum>
                </property>
                <property name="placeholderText">
                 <string>password</string>
                </property>
                <property name="clearButtonEnabled">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="1" column="2">
               <widget class="QLineEdit" name="le_Username">
                <property name="maxLength">
                 <number>40</number>
                </property>
                <property name="placeholderText">
                 <string>user name or id</string>
                </property>
                <property name="clearButtonEnabled">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="0" column="2">
               <widget class="QLabel" name="lbl_DatabaseName">
                <property name="text">
                 <string>Database URL will go here</string>
                </property>
                <property name="openExternalLinks">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="pg_Logoff">
          <layout class="QVBoxLayout" name="vl_PageLogoff">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QFrame" name="fr_Logoff">
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QGridLayout" name="gl_Logoff">
              <property name="leftMargin">
               <number>4</number>
              </property>
              <property name="topMargin">
               <number>4</number>
              </property>
              <property name="rightMargin">
               <number>4</number>
              </property>
              <property name="bottomMargin">
               <number>4</number>
              </property>
              <property name="spacing">
               <number>6</number>
              </property>
              <item row="4" column="1" alignment="Qt::AlignRight">
               <widget class="QPushButton" name="pb_Logoff">
                <property name="text">
                 <string>logoff</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QPlainTextEdit" name="te_Roles">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="documentTitle">
                 <string>User roles</string>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="lbl_Roles">
                <property name="text">
                 <string>Roles:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="lbl_Name">
                <property name="text">
                 <string>Name:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="le_Name">
                <property name="readOnly">
                 <bool>true</bool>
                </property>
                <property name="placeholderText">
                 <string>user name</string>
                </property>
                <property name="clearButtonEnabled">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QLineEdit" name="le_Info">
                <property name="readOnly">
                 <bool>true</bool>
                </property>
                <property name="placeholderText">
                 <string>info text will go here</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="lbl_Info">
                <property name="text">
                 <string>Info:</string>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <spacer name="hs_Logoff">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Fixed</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>75</width>
                  <height>10</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="3" column="1">
               <spacer name="vs_Logoff">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tb_LoginToDbHints">
      <attribute name="title">
       <string>Login hints, trouble shooting</string>
      </attribute>
      <layout class="QVBoxLayout" name="vl_LoginHints">
       <property name="leftMargin">
        <number>4</number>
       </property>
       <property name="topMargin">
        <number>4</number>
       </property>
       <property name="rightMargin">
        <number>4</number>
       </property>
       <property name="bottomMargin">
        <number>4</number>
       </property>
       <item>
        <widget class="QTextBrowser" name="tbr_InfoAndHints">
         <property name="documentTitle">
          <string/>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.1pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;ol style=&quot;margin-top: 0px; margin-bottom: 0px; margin-left: 0px; margin-right: 0px; -qt-list-indent: 1;&quot;&gt;&lt;li style=&quot; font-size:8pt;&quot; style=&quot; margin-top:12px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;You can go directly to the swift DB: ##swiftDb## &lt;/li&gt;
&lt;li style=&quot; font-size:8pt;&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;If you want to use VATSIM SSO, you need to enable it once ##swiftEnableSSO## &lt;/li&gt;
&lt;li style=&quot; font-size:8pt;&quot; style=&quot; margin-top:0px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;In case you do not have a password you can try &amp;quot;anonymous&amp;quot;/&amp;quot;anonymous&amp;quot;. No guarantee it will work, but worth a trial. &lt;/li&gt;&lt;/ol&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::TextBrowserInteraction</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tb_LoginToDbAndHints</tabstop>
  <tabstop>le_Username</tabstop>
  <tabstop>le_Password</tabstop>
  <tabstop>pb_Login</tabstop>
  <tabstop>tbr_InfoAndHints</tabstop>
  <tabstop>pb_Logoff</tabstop>
  <tabstop>le_Name</tabstop>
  <tabstop>le_Info</tabstop>
  <tabstop>te_Roles</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
